products/kubernetes-csi-node-driver-registrar.md

```diff
@@ -0,0 +1,105 @@
+---
+title: kubernetes-csi-node-driver-registrar
+category: app
+tags: kubernetes
+permalink: /kubernetes-csi-node-driver-registrar
+alternate_urls:
+-   /kubernetes-csi-node-driver-registrar
```

Comment:
These are for redirects, so we don't need the exact match, but rather alternative versions of the URL that users might type:

```suggestion
-   /k8s-csi-node-driver-registrar
-   /node-driver-registrar
```

Reply:
thanks so much!